lib/k8sClient.js

```diff
@@ -2,6 +2,8 @@
 
 const k8s = require('k8s'),
     debug = require('debug')('helmit:k8sClient');
+var STRICT_SSL = process.env.STRICT_SSL || 'doc,iso';
+var AUTH_TOKEN = process.env.AUTH_TOKEN || '';
```

Comment:
Should be using either `const` or `let` instead of `var`. In this case, I'd probably go with `const`.

Also, no reason to have the keyword, as you can assign multiple values with one keyword using commas as delimiters.

```js
const k8s = require('k8s'),
    debug = require('debug')('helmit:k8sClient'),
    STRICT_SSL = process.env.STRICT_SSL || 'doc,iso',
    AUTH_TOKEN = process.env.AUTH_TOKEN || '';
```